Makefile

```diff
@@ -56,7 +56,6 @@ install-gen-deps: ## Install dependencies for code generation
 	# out code dependencies; so a go mod tidy will remove them again. `go
 	# install` seems to install the last tagged version and we want to install
 	# master.
-	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/mna/pigeon@master)
```

Comment:
Do we want to put that one in the sdk folder for now ?

Reply:
I've got the new sdk makefile locally in the sdk repo, but I can put it in the sdk folder.